File: Array/Array.cpp
#include<iostream>
using namespace std;
int main(){
    int size;
    cout<<"Enter the size of the array :";
    cin>>size;
    int item;
    int arr[size];
    for(int i = 0; i<size ; i++){
        cout<<"Enter the Element of Array :";
        cin>>item;
        arr[i]=item;
    }

    cout<<"Array is : ";
     for(int j = 0; j<size ; j++){
        cout<<arr[j];
    }


}

File: Array/Array.py
size = int(input('Enter the size of the array : '))
arr = []
print("Enter "+ str(size) + " elements of the array :")
for i in range(size):
    item = int(input("Enter the"+str(i+1)+ "element :"))
    arr.append(item)
        
print("Array is :",arr)